website/static/js/fangorn.js

```diff
@@ -1061,12 +1654,35 @@ tbOptions = {
         $(document).on('click', '.fangorn-dismiss', function() {
             tb.redraw();
         });
+        tb.select('#tb-tbody').on('click', function(event){
+            if(event.target !== this) {
+                return;
+            }
+            tb.clearMultiselect();
+            _fangornResetToolbar.call(tb);
+        })
 
         $(window).on('beforeunload', function() {
             if (tb.dropzone && tb.dropzone.getUploadingFiles().length) {
               return 'You have pending uploads, if you leave this page they may not complete.';
             }
         });
+        if(tb.options.placement === 'project-files') {
+            _resizeHeight.call(tb);
+            $(window).resize(function(){
+                _resizeHeight.call(tb);
+            })
+        }
+        $(window).on('keydown', function(event){
+            if (event.keyCode === 27) {
+                _fangornResetToolbar.call(tb);
+            }
+        });
+        $(document).on('keypress', '#createFolderInput', function () {
```

Comment:
Mithril has a problem with directly running functions in the middle of its view like onKey is doing. The best you can do is bind which is still doable.

And as for adding them to the view jquery is helping with cross browser compatibility with its own keycode variable. 

What is the reason to not use jquery in this case?